u2f-tests/NFC/u2f_nfc_util.c

```diff
@@ -24,9 +24,17 @@ flag arg_Pause = flagOFF;
 flag arg_Abort = flagON;
 cmd_apdu_type cmd_apdu;
 
+//Chaining Blocksize frpm reader - Le
+static uint16_t blockSize = 256;
+
 //Shared between PC/SC Card Access Routines
 static SCARDHANDLE hCard;
 
+void setChainingLc(uint16_t size){
+  blockSize = (size<=256?size:256);
```

Comment:
Not sure I understand the comment – blockSize is the blocksize from reader.

It’s a short so that it can contain values from  0 to 256 (inclusive) and a function to set it to a required value, limited to a max of 256. I don’t understand the reuse comment?

Regards - Chris

From: Alexei [mailto:notifications@github.com]
Sent: Wednesday, December 02, 2015 11:22 AM
To: google/u2f-ref-code
Cc: Woodthorpe Christopher (IFAM CCS SMD AMR)
Subject: Re: [u2f-ref-code] Added Kh size tests (#62)


In u2f-tests/NFC/u2f_nfc_util.c<https://github.com/google/u2f-ref-code/pull/62#discussion_r46461200>:

>  //Shared between PC/SC Card Access Routines

>  static SCARDHANDLE hCard;

>

> +void setChainingLc(uint16_t size){

> +  blockSize = (size<=256?size:256);

please add whitespace. Also, why not reuse blockSize here?

—
Reply to this email directly or view it on GitHub<https://github.com/google/u2f-ref-code/pull/62/files#r46461200>.